src/scss/components/_mobile-scroll-carousel.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * Mobile scroll carousel
+ * --------------------------------
+ * Used for mobile filter carousel
+ *
+ * ================================= */
+
+.c-mobileScrollCarousel {
```

Comment:
Same with this – best to name it with respect to it being a scroll component as it could be used on wider viewports potentially.